File: business_logic/Model/Mediator/Comunication.cs
namespace business_logic.Model.Mediator
{
    public class Comunication<T>
    {
        public string type {get;}
        public string method {get;}
        public T value {get;}
        public Comunication (string type, string method, T value){
            this.type = type;
            this.method = method;
            this.value = value;
        }
    }
}

File: database_conection/src/mediator/Comunication.java
package mediator;

public class Comunication<T>
{
  private String type;
  private String method;
  private T value;

  public Comunication(String type, String method, T value)
  {
    this.type = type;
    this.method = method;
    this.value = value;
  }

  public String getType()
  {
    return type;
  }

  public String getMethod()
  {
    return method;
  }

  public T getValue()
  {
    return value;
  }

}
